crates/lang/codegen/src/generator/dispatch.rs

```diff
@@ -706,7 +732,6 @@ impl Dispatch<'_> {
             );
             let accepts_payment = quote_spanned!(message_span=>
                 false ||
-                !#any_message_accept_payment ||
```

Comment:
Can we have a UI test for this: Multiple constructors which are all not payable?